capten/config-worker/internal/crossplane/config_cluster_updates.go

```diff
@@ -305,3 +321,35 @@ func readClusterDefaultApps(clusterDefaultAppsFile string) ([]DefaultApps, error
 
 	return appList.DefaultApps, nil
 }
+
+func getConfigGlobalValues(ctx context.Context) (map[string]string, error) {

```

Comment:
use GetClusterGlobalValues in common-pkg/credential/client.go.
this is yaml file, decode as yaml and convert to map[string]string